lib/MixedModels/LMM.rb

```diff
@@ -229,6 +275,8 @@ def LMM.from_formula(formula:, data:, weights: nil, offset: 0.0, reml: true,
   # The random effects are specified as an Array of Arrays, where the variables in each
   # sub-Array correspond to a common grouping factor (given in the Array +grouping+) and are 
   # modeled as correlated.
+  #Since elements of random_effects correspond to elements of grouping, maybe you
+  #could combine these two together with a hash or small utility class?
```

Comment:
Yes, I think that would be more natural. Maybe something like 
`random_effects: {terms: [[:x, :y], [:z]], grouping: [:v, :w]}`